must-gather/collection-scripts/gather_operator

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+OSC_PATH=${BASE_COLLECTION_PATH}/sandboxed-containers/operator
```

Comment:
Rather than using this, you should take advantage of what already exists in the code, as in https://github.com/openshift/sandboxed-containers-operator/blob/0c4e7aaab1df9ae92b66b28ceb1528eab9c7bd23/must-gather/collection-scripts/gather_sandboxed_containers_logs#L23

There we already create the namespace for logs we collect from the operators that are interesting for us.